test/main/store/migrations/index.test.js

```diff
@@ -1475,3 +1488,89 @@ describe('migration 35', () => {
     })
   })
 })
+
+describe('migration 37', () => {
+  beforeEach(() => {
+    state.main._version = 36
+  })
+
+  describe('when altGr is a modifier', () => {
+    beforeEach(() => {
+      state.main.shortcuts = {
+        summon: { modifierKeys: ['AltGr'], shortcutKey: 'Slash', enabled: true, configuring: false }
+      }
+    })
+
+    it('should update the summon shortcut on Linux', () => {
+      withPlatform('linux', () => {
+        const updatedState = migrations.apply(state, 37)
+        const { shortcuts } = updatedState.main
+
+        expect(shortcuts).toStrictEqual({
+          summon: {
+            modifierKeys: ['Alt'],
+            shortcutKey: 'Slash',
+            enabled: true,
+            configuring: false
+          }
+        })
+      })
+    })
+
+    it('should update the summon shortcut on Windows', () => {
+      withPlatform('win32', () => {
+        const updatedState = migrations.apply(state, 37)
+        const { shortcuts } = updatedState.main
+
+        expect(shortcuts).toStrictEqual({
+          summon: {
+            modifierKeys: ['Alt', 'Control'],
```

Comment:
is there a case on either platform where either of these modifiers could have already been in the list and we're left with a list like `['Alt', 'Control', 'Control']`?

Reply:
Hmmm possibly, I'll address this.

Reply:
I verified this on Linux, I couldn't get `AltGr` + `Alt` + ... to register - as soon as you press the left Alt key it overrides the `AltGr`.